using_images/other_images/jenkins.adoc

```diff
@@ -52,9 +52,24 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
 
 === Initializing Jenkins
 
+ifdef::openshift-origin[]
```

Comment:
why is this ifdef'd to origin? it applies to OCP too right?  (starting with v3.4 so you'll need to label the PR appropriately)